R/repr_htmlwidget.r

```diff
@@ -22,3 +22,57 @@ repr_html.htmlwidget <- function(obj, ...) {
 	
 	readChar(htmlfile, file.info(htmlfile)$size)
 }
+
+
+
+# sort of like htmlwidgets::saveWidget(), but for HTML tags
+save_html <- function(obj, ...) {
```

Comment:
the name isn’t very truthful, as the saving is just a side effect.

could you just name it `repr_html.shiny.tag` and then do `repr_html.shiny.tag.list <- repr_html.shiny.tag`?